home/migrations/0050_sitesettings_maximum_width.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.14 on 2022-11-25 07:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('home', '0049_auto_20221018_0943'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='sitesettings',
+            name='maximum_width',
```

Comment:
@cbunicef it's updated. Can you check please?

Reply:
The Wagtail documentation is giving instruction on how to use the `image` tag, rather than advice on how to name fields. I agree that the field name should be more descriptive than `maximum_width`, but `image_maximum_width` is probably good enough.